gf-webapp/src/model/Price.ts

```diff
@@ -5,12 +5,12 @@ export class Price {
   /**
    * Price currency
    */
-  private currency: Currency;
+  public currency: Currency;
```

Comment:
Why do u change modifications from private to public?

Reply:
Return to private

Reply:
Back to the public fields because autogenerated model interfaces couldn't has private fields

Reply:
So, do u think u can't use private fields in any business models then?

Reply:
toJSON() {}

Reply:
Done

Reply:
Done